### PR TITLE
Feature/auto navigation

### PR DESCRIPTION
**Pull request by andij**
Initial attempt at a solution for providing auto navigation for the Prototype tool.

Story ticket: https://github.com/ConnectedHomes/nucleus/issues/2178

### CODE REVIEW

File: src/_layouts/base.njk [hunk 1]
```diff
@@ -11,6 +11,10 @@ build: www5
     <meta name="description" content="{{title}}">
     <meta name="format-detection" content="telephone=no">
 
+    <link href="/assets/styles.css" rel="stylesheet">
+    <script type="text/javascript" src="/assets/treeLinks.js"></script>
+    <script type="text/javascript" src="/assets/treeitemLinks.js"></script>
```
Comment (jholt1) [inline src/_layouts/base.njk:16]: Are these needed?

File: src/assets/treeLinks.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
+/*
```
Comment (jholt1) [inline src/assets/treeLinks.js:1]: Is this needed?

File: src/assets/treeitemLinks.js [hunk 3]
```diff
@@ -0,0 +1,267 @@
+/*
```
Comment (jholt1) [inline src/assets/treeitemLinks.js:1]: Is this needed?

File: src/_layouts/base.njk (hunk 1 above)
Comment (andij) [reply src/_layouts/base.njk:16]: They are included to introduce the accessibility attributes, switching the tabindex from -1 to 0 etc.
Also adding the keyboard navigation to the navtree.
As described here: https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/treeview-2a.html

**Review by jholt1 — APPROVED** [review]

**Pull request merged**